lib/OpenQA/Script/CloneJob.pm

```diff
@@ -327,7 +327,7 @@ sub clone_job ($jobid, $url_handler, $options, $post_params = {}, $jobs = {}, $d
     if (my $group_id = $job->{group_id}) { $settings->{_GROUP_ID} = $group_id }
     clone_job_apply_settings($options->{args}, $relation eq 'children' ? 0 : $depth, $settings, $options);
     OpenQA::Script::CloneJobSUSE::detect_maintenance_update($jobid, $url_handler, $settings)
-      unless $options->{'skip-checks'};
+      if $options->{'check-repos'};
```

Comment:
@grisu48 you first introduced this in e0ebe7648. I assume you wanted to introduce a generic option to skip *any* checks including any potential future additions? In that case we could keep the old option and only introduce a new option `check-repos`. The old option would then of course not do anything useful so I guess @asmorodskyi's change is fine but I would just like to give you the opportunity to intervene

Reply:
Like I mention in commit message having option `--skip-checks`  which is doing single check is confusing for **current** users . When I read parameter name and its description I really like to know what it is doing **currently** not that someone will potentially implement in it in 5 years

Reply:
I suggest keeping `--skip-checks` for compatibility but with a deprecation message.

The user story is not fully clear to me, though, so I'm a little worried about introducing a new option for a feature that isn't widely used (anymore)... do we actually want to keep it? Can we please confirm the use case before adding more new options here?

Reply:
well my opinion that you are OVERestimating that . We are not breaking kernel API or glibc . It is script which is in reality used in 5-10 different places in non-interactive way . And half of this places are under control of people who currently reviewing this PR . Also whoever will be affected by this break will fix it in 5 minutes :man_shrugging:  So even effort of this discussion around this deprecation is bigger then actual impact.
But let's stop here I would just do whatever you will tell me

Reply:
It might only take 5 minutes for the actual fix, but it might break pipelines and things are blocked until someone can actually do the fix.
When I recently did the perltidy change, there was a big complaint that this was blocking osado PRs, although the fix was just running perltidy.
If you are in the position of the person whose pipelines breaks / is blocked you might think differently.

Reply:
just to clarify : all the noise around perltidy was about the setup which lead to flow where change in one project may break another one and not about change itself .
About estimation how much disruptive this change may be I just share my view on that . Didn't want to convince you . Anyway you are "gatekeepers"  of this project so I will just follow whatever you saying

Reply:
> do we actually want to keep it? Can we please confirm the use case before adding more new options here?

Note I filed [poo#182834 regarding follow-up work](https://progress.opensuse.org/issues/182834), so no strong objection to introducing `--check-repos` here.